reco_utils/recommender/fastai/fastai_utils.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+
+import numpy as np
+import pandas as pd
+import fastai
+from fastprogress import force_console_behavior
+import fastprogress
+
+
+def cartesian_product(*arrays):
```

Comment:
why not put this in a more general location?

Reply:
afaik, we are not using this in any other notebook or algo, but tbh, I don't have a strong opinion on this

Reply:
added docstrings and explanation here